### PR TITLE
app-admin/syslog-ng: add 4.6.0, drop old

### PR DESCRIPTION
**Pull request by hydrapolic**
Copy of previous ebuild, tests pass on ~amd64

```
============================================================================
Testsuite summary for syslog-ng 4.6.0
============================================================================
# TOTAL: 184
# PASS:  184
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
>>> Completed testing app-admin/syslog-ng-4.6.0
```

### CODE REVIEW

**Review by arkamar — CHANGES_REQUESTED** [review]

File: app-admin/syslog-ng/syslog-ng-4.6.0.ebuild [hunk 1]
```diff
@@ -3,31 +3,35 @@
...
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{10..12} )
 inherit autotools python-single-r1 systemd
 
-MY_PV_MM=$(ver_cut 1-2)
 DESCRIPTION="syslog replacement with advanced filtering features"
 HOMEPAGE="https://www.syslog-ng.com/products/open-source-log-management/"
 SRC_URI="https://github.com/balabit/syslog-ng/releases/download/${P}/${P}.tar.gz"
```
[the remark this answers is not in the document]
Comment (hydrapolic) [reply app-admin/syslog-ng/syslog-ng-4.6.0.ebuild]: Hello @arkamar, sure, updated, thanks for the notice.

**Comment by Flowdalic** [comment]
@hydrapolic thanks for your contribution.

Could you may look into the automagic MQTT dependency of syslog-ng ([bug #92201](https://bugs.gentoo.org/922201)). I wouldn't be surprised if there are further automagic dependencies.

**Comment by thesamesam** [comment]
I'll merge it now given https://bugs.gentoo.org/914364.

**Comment by hydrapolic** [comment]
Thanks @Flowdalic for the bug, didn't get to it as of now. Thanks @thesamesam for the help too.

**Pull request closed without merging**